src/components/CallToAction.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="call-to-action">
+    <div class="call-to-action--thumb">
```

Comment:
.call-to-action--thumb & --description are both `Elements` according to BEM rules not `Modifiers`. Should be .call-to-action__thumb && __description.